src/busctl/busctl.c

```diff
@@ -2331,6 +2418,8 @@ static int help(void) {
                "                           Call a method\n"
                "  emit OBJECT INTERFACE SIGNAL [SIGNATURE [ARGUMENT...]]\n"
                "                           Emit a signal\n"
+               "  wait OBJECT INTERFACE SIGNAL\n"
```

Comment:
i think doing it like this is OK, but note that dbus has its own matching language, i guess sooner or later people want to use that. But I guess we can add that via a switch later, if we really want to.

Reply:
(I think the dbus match language is a bit overdesigned, hence I think the simple triplet you are matching against here is a lot nicer, in particular for completion)